thorlcr/thorutil/thmem.cpp

```diff
@@ -1237,6 +1237,8 @@ CThorSpillableRowArray::CThorSpillableRowArray(CActivityBase &activity)
     : CThorExpandingRowArray(activity)
 {
     initCommon();
+    commitDelta = CommitStep;
+    throwOnOom = false;
```

Comment:
There's another constructor below where these are not set...

Using member initialization syntax is probably better...